### ISSUE TITLE
Docker support

### ISSUE THREAD

**Issue by P-Storm** [koenvzeijl/AspNetCore.SassCompiler#44]
Currently the code below is throwing an error (as is logical) in a docker container.

```c#
builder.Services.AddSassCompiler();
```

Maybe add a note that it isn't supported in docker container (also discovered that hot reload isn't standard enabled in a docker container).

**Comment by sleeuwen** [comment]
Hi, thanks for reporting this, I don't work much with docker so I wasn't aware that it didn't work. I will definitely add a note to the readme about that.

I understand it you're using a docker container during development, correct? I think it's worth investigating if we can make it work for that scenario, but I don't have much experience with developing in a docker container so I'll have to look into it to see what we can do.

**Comment by koenvzeijl** [comment]
@P-Storm Could you try to check if your issue has been resolved with the latest version?

**Comment by koenvzeijl** [comment]
@P-Storm I will close this for now. If the problem still occurs, feel free to open this again

**Comment by Neurion** [comment]
While `builder.Services.AddSassCompiler();` doesn't throw errors for me in a dev container, the watcher doesn't work. I have to restart `dotnet watch` for the SASS to compile. I'm using version 1.80.4 of the NuGet package.
